examples/nek5000/make-nek-examples.sh

```diff
@@ -49,7 +49,13 @@ if [[ "$#" -eq 1 && "$1" -eq "clean" ]]; then
   exit 0
 fi
 
-: ${NEK5K_DIR:=`cd "../../../Nek5000"; pwd`}
+# Set defaults for the parameters
+if [ ! -f $NEK5K_DIR/bin/makenek ]; then
+  NEK5K_DIR:=`cd "../../../Nek5000"; pwd`
```

Comment:
This isn't bash syntax.
```
$ ./make-nek-examples.sh                                                                                                               
./make-nek-examples.sh: line 54: NEK5K_DIR:=/home/jed/src/Nek5000: No such file or directory                                                                                                  
cp: cannot stat '/bin/makenek': No such file or directory                                                                                                                                     
sed: can't read makenek: No such file or directory                                                                                                                                            
Building example: ex1 ...                                                                                                                                                                     
./make-nek-examples.sh: line 87: ./makenek: No such file or directory                                                                                                                         
  Building ex1 failed. See ex1.build.log for details.             
```

and besides, I prefer the original behavior (providing a default value). Adding a test to error if that doesn't work would be great.

Reply:
Hmm... the old version didn't work for me:
```sh
./make-nek-examples.sh: line 54: cd: ../../../Nek5000: No such file or directory
./make-nek-examples.sh: line 54: NEK5K_DIR:=/home/kolev1/ceed/libCEED/examples/nek5000: No such file or directory
Building example: ex1 ...
  Building ex1 failed. See ex1.build.log for details.
```
Does
```
if [ ! -e $NEK5K_DIR/bin/makenek ]; then
```
work for you?

Reply:
Pretty sure that error message is coming from the "new" version that you just committed. I would write

    if [ -e ${NEK5K_DIR:=`cd "../../../Nek5000"; pwd`} ]; then
        echo "Failed to find Nek5000 at ${NEK5K_DIR}"
        exit 1
    fi